src/endpoints/refreshEndpoint.ts

```diff
@@ -34,7 +34,9 @@ export const refresh = (
     const id = hpkeKeyIdMap.size + 1;
 
     // Generate HPKE key pair with a six digit id
```

Comment:
the comment is not wrong

Reply:
six digit id?

Reply:
I will fix this, it should be just 2 digit id